blogs/alexis.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://cdn.tailwindcss.com"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" referrerpolicy="no-referrer" >
+    <link rel="stylesheet" href="./blog.css">
+    <title>Blog: Alexis</title>
+
+</head>
+<body>
+    <header class="w-full h-20 top-0 z-20 bg-slate-300 fixed">
+        <div class="w-full h-full justify-between flex flex-cols items-center py-0 px-0 xl:px-[1.5rem] ">
+            <a href="../blog.html" class="mx-4 w-12 h-12"><img src="./img/back.png" alt=""></a>
+            <div class="logo"><a class="logo-border" href="#">
+                <img class="rounded-full logo-size w-16 h-16" src="../Assets/Images/logo.jpg" alt="logo" />
+              </a></div>
+              <a href="../index.html" class="mr-10 ml-4 w-12 h-12"><img src="./img/home.png" alt=""></a>
+            </ul>
+        </div>
+    </header>
+
+    <div class=" w-[32rem]  xl:w-[117rem] my-0 mx-0 xl:mx-auto">
+        <div class="w-[360px] xl:w-[114rem] h-[25rem] xl:h-[40rem] my-[5rem] xl:mx-auto bg-[url('./img/b12-poster.png')] bg-no-repeat bg-cover snap-center rounded-xl">
+
+        </div>
+
+        <main class="w-full bg-white p-5 xl:p-12 rounded-lg -mt-56 items-center my-0 mx-[20px] xl:mx-auto xl:w-[80rem]">
+            <h2 class="text-3xl xl:text-5xl font-semibold uppercase tracking-wider">Rescue Rooster</h2>
+            <div class="flex justify-between items-center my-[3rem] mx-0">
+                <div class="flex justify-between items-center">
+                    <div class="w-24 h-24">
+                        <a href="https://littlemissbuttons.github.io">
+                            <img src="https://littlemissbuttons.github.io/images/Profile-Beenie.jpg" class="rounded-3xl">
+                        </a>
+                    </div>
+                    <div class="text-gray-700 text-xl font-normal my-0 mx-[1rem] tracking-normal">
+                        <a href="https://littlemissbuttons.github.io">
+                            <h3 class="uppercase"><b>Alexis Pendleton</b></h3>
+                            <p>October 27, 2022</p>
+                        </a>
+                    </div>
+                </div>
+            </div>
+            <div class="content">
+                <p class="text-[#5b5963] my-[1rem] mx-0 text-xl">‘Has anyone lost a chicken? We’ve one at the shop!’
```

Comment:
```suggestion
                <h3 class="text-xl xl:text-2xl font-semibold tracking-wide">‘Has anyone lost a chicken? We’ve one at the shop!’</h3>
```

Do this changes in all your heading points @LittleMissButtons

Reply:
@LittleMissButtons  Otherwise its all okay.. please quickly change this.

Reply:
Hi, thanks for the feedback and I have updated that now and pushed the changes.

Hopefully that comes through ok. Please could you add the Hacktoberfest-Accepted label to this PR. Cheers